### PR TITLE
keyctl: add page

### PR DESCRIPTION
**Pull request by RaitoBezarius**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** `keyctl from keyutils-1.6.3`

### CODE REVIEW

**Comment by sbrl** [comment]
Sorry for the delay, @RaitoBezarius! We have an enormous number of PRs to handle at the moment due to Hacktoberfest. We usually have between 20 and 30 PRs open at a time, but at the moment we have well over 100.....

**Review by sbrl — APPROVED** [review]
Thanks, @RaitoBezarius!

File: pages/linux/keyctl.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- List current keys in the user default session:
+
+`keyctl list {{@us}}`
+
+- Store a key in a specific keyring:
+
+`keyctl add {{type_keyring}} {{key_name}} {{key_value}} {{target_keyring}}`
+
+- Store a key with its value on the standard input:
```
Comment (CleanMachine1) [inline pages/linux/keyctl.md:18]: Could you change `on` to `from` and remove the word `the`

File: pages/linux/keyctl.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`echo -n {{key_value}} | keyctl padd {{type_keyring}} {{key_name}} {{target_keyring}}`
+
+- Put a timeout on a key:
+
+`keyctl timeout {{key_name}} {{timeout_in_seconds}}`
+
+- Read a key and formats it as a hex-dump if not printable:
```
Comment (CleanMachine1) [inline pages/linux/keyctl.md:26]: Remove the `s` from `formats`

File: pages/linux/keyctl.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`keyctl read {{key_name}}`
+
+- Read a key and formats as-is:
```
Comment (CleanMachine1) [inline pages/linux/keyctl.md:30]: Again remove the plural s

**Review by CleanMachine1 — COMMENTED** [review]
Sorry for the lack of the suggestion feature, I'm on mobile and there isn't an easy option :(

**Comment by RaitoBezarius** [comment]
Thanks @sbrl no worries I do understand the context :)

@CleanMachine1 the comments are addressed :)

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**